dmoj/executors/NODEJS.py

```diff
@@ -0,0 +1,44 @@
+from dmoj.executors.script_executor import ScriptExecutor
+from dmoj.cptbox.filesystem_policies import ExactFile
+
+
+class Executor(ScriptExecutor):
+    ext = 'js'
+    command = 'node'
+    nproc = -1
+    command_paths = ['node', 'nodejs']
+    syscalls = [
+        # Start syscalls copied from COFFEE.py
+        'capget',
+        'eventfd2',
+        'newselect',
+        'sched_yield',
+        'select',
+        'setrlimit',
+        'statx',
+        # End syscalls copied from COFFEE.py
+        
+        # Without this, we get
+        # > Protection fault on: 425 (sys_io_uring_setup)
+        'io_uring_setup',
```

Comment:
Can we try making this return `ENOSYS`? I don't think `io_uring` at this time due to potential security implications.

Reply:
Good call: I was able to remove this by setting an environment variable to disable this:

```
    def get_env(self):
        env = super().get_env()
        # Disable io_uring due to potential security implications
        env['UV_USE_IO_URING'] = '0'
        return env
```